tests/testthat/test-print.R

```diff
@@ -30,9 +30,6 @@ test_that("print.coverage prints by = \"line\" by default", {
 
   expect_message(print(cov),
     rex::rex("R/TestPrint.R: ", anything, "0.00%"))
-})
-test_that("print.coverage prints by = \"line\" by default", {
-  cov <- package_coverage("TestPrint")
```

Comment:
Could remove this and have it as the same test as previous, since they rely on the same package coverage. Allows to compute coverage only once.